Client/src/Components/GroupedTask/GroupedTask.jsx

```diff
@@ -127,25 +130,6 @@ export default function GroupedTask(props){
     )}
   })
 
-  // console.log('taskElementArrays', taskElementArrays)
```

Comment:
It looks like this section didn't make it through the merge conflict when you merged the master branch into your branch, but it is necessary for calculating the elapsed times of each task.  Could you copy and paste the code from here into your GroupedTask.jsx file then make a new commit and pull request?